translations/en.json

```diff
@@ -0,0 +1,22 @@
+{
+  "meta.title": "Eresource Finder Plugin",
```

Comment:
Labels in Agreements all seem to have a hyphen between the "e" and "r" in eresource.